package.json

```diff
@@ -51,19 +51,27 @@
         "onCommand:staticWebApps.refresh",
         "onCommand:staticWebApps.reportIssue",
         "onCommand:staticWebApps.selectSubscriptions",
-        "onCommand:staticWebApps.setupRunningInVSCode",
         "onCommand:staticWebApps.showActions",
         "onCommand:staticWebApps.staticWebApps.action.cancel",
         "onCommand:staticWebApps.staticWebApps.action.rerun",
         "onCommand:staticWebApps.uninstallStaticWebAppsCli",
         "onCommand:staticWebApps.viewProperties",
+        "onDebug",
```

Comment:
Would `onDebugResolve:swa` work here?

Reply:
I removed `onDebug` and I don't think we need `onDebugResolve:swa` since we should only ever provide and therefore have to resolve debug configs with the type `pwa-chrome`.

The only reason I'm contributing our own debug type is so we can have our own menu item here:
![image](https://user-images.githubusercontent.com/12476526/139725097-7014ac53-998c-4620-8b6f-706f45b0d7a8.png)

Otherwise it would show up under a generic label like Chrome or Node.js I think.

Reply:
But we have a `resolveDebugConfigurations` implemented, doesn't that mean we need to support `onDebugResolve`? And thus should have the activation event?

Reply:
Follow up: What I tried to do, and what I still want to do if it's possible is make the `swa` debug type an "alias" for `pwa-chrome`. I ran into some issues and just ended up only using `pwa-chrome`.

Reply:
> But we have a resolveDebugConfigurations implemented, doesn't that mean we need to support onDebugResolve? And thus should have the activation event?

We have `onDebugResolve:pwa-chrome`, is there a reason this isn't enough?

Reply:
Ah! Now I understand what you mean. Yeah, you got it right, this should work.

Reply:
I know that this is already merged, but we do want to natively support edge/mozilla debugging as well?  We should be able to just see if they have VS Code debugger extensions installed or not.

Reply:
I believe `pwa-edge` is built-in via https://github.com/microsoft/vscode-js-debug. So we'd need to provide a setting for users to control which they prefer, edge or chrome.

How important is this? Is it worth the added complexity to support the other browsers?

Reply:
Right now, I'd lean towards "no" though I do think it's safer to assume `pwa-edge` since it's built in, as you mentioned.

I guess I am a little concerned if we would get criticism for forcing users on a "Microsoft" path especially since the tasks are dynamic and they wouldn't be able to easily change it.

A check to see if they have the chrome debugger might be worth it though because it'll just fail if they don't have it installed, right?

Reply:
I think it's actually built-in, the Chrome Debugger extension is deprecated in favor of the built-in ones.

See deprecated Chrome debugger: https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome

Reply:
Ok, disregard then